modules/simple_away.cpp

```diff
@@ -204,6 +227,11 @@ class CSimpleAway : public CModule {
 		m_iAwayWait = iAwayWait;
 	}
 
+	void SetMinClients(unsigned int iMinClients, bool bSave = true) {
```

Comment:
It would be nice if it were rejected when user input was malformed. i.e. `if (iMinClients == 0 || iMinClients > 100) { ... }`, it shouldn't actually change anything but I think it'd add to the polish.